### ISSUE TITLE
Multiple class diagrams are not rendered correctly

### ISSUE THREAD

**Issue by tmatz** [mermaid-js/mermaid#1044]
**Describe the bug**

When there are multiple class diagrams in a same page, diagram's layout break.

**To Reproduce**

Following markdown code:

````markdown
```mermaid
classDiagram
    Class01 <|-- Cool
```

```mermaid
classDiagram
    Class01 <|-- Cool
```
````

Result is this (Visual Studio Code + Markdown Preview Mermaid Support extension):

![image](https://user-images.githubusercontent.com/4199487/68015799-57568680-fcd6-11e9-997f-63a57764e262.png)

**Expected behavior**

Second diagram should be same as the first diagram.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows10
 - Browser: Visual Studio Code (1.39.2)
 - Version: Markdown Preview Mermaid Support(1.4.0 uses mermaid 8.4.0)

**Comment by Dunning-Kruger** [comment]
I believe this is not an issue with the library but of how the extension uses it.
If you have any extra information, we'd be happy to help.

**Comment by tmatz** [comment]
Unfortunately, this is bug of this library.

I already have fixed this bug in my clone repository, and confirmed the vscode extension works correctly.

I'm looking how to implement test for multiple diagram.

This bug seems have fixed once at #832, but re-introduced by #795.

**Comment by Dunning-Kruger** [comment]
@tmatz You could do an e2e test. Check [the guide](https://github.com/mermaid-js/mermaid/blob/master/CONTRIBUTING.md) for some pointers . People might also be able to help you out or point you in the right direction [on slack](https://join.slack.com/t/mermaid-talk/shared_invite/enQtNzc4NDIyNzk4OTAyLWVhYjQxOTI2OTg4YmE1ZmJkY2Y4MTU3ODliYmIwOTY3NDJlYjA0YjIyZTdkMDMyZTUwOGI0NjEzYmEwODcwOTE).
